hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestSnapshotDeletingService.java

```diff
@@ -188,6 +191,30 @@ public void testMultipleSnapshotKeyReclaim() throws Exception {
     assertTableRowCount(deletedTable, 0);
 
     verifySnapshotChain(delSnapInfo, null);
+
+    // verify the cache of purged snapshot
+    // /vol1/bucket2/bucket2snap1 has been cleaned up from cache map
+    SnapshotCache snapshotCache = om.getOmSnapshotManager().getSnapshotCache();
+    Method getDbMapMethod = snapshotCache.getClass()
```

Comment:
I think you can directly use [SnapshotCache#size()](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotCache.java#L94).

Reply:
oh! you're absolutely right! let me change it!